packages/api/src/utils/db-client.js

```diff
@@ -554,19 +563,66 @@ export class DBClient {
         'uploads_multipart_total',
       ])
 
-    if (error) {
-      throw new DBError(error)
-    }
+    const dagByteSizeQuery = metricsQuery
+      .select('name, dimensions, value, collected_at')
```

Comment:
We don't use the `collected_at` value.

```suggestion
      .select('name,dimensions,value')
```

Reply:
fixed